pages/plant_memory.py

```diff
@@ -0,0 +1,335 @@
+            "stop": [0.53, 14, 20 + slider_time, 30 + slider_time],
+            "color": ["#1c5bc7", "#cf6d0c", "#1c5bc7", "#d10a0d"],
+        }
+    )
+
+    areas = (
+        alt.Chart(areas_data.reset_index())  # type: ignore
+        .mark_rect(opacity=0.3)  # type: ignore
+        .encode(
+            x2="stop",
+            y=alt.value(0),
+            y2=alt.value(290),  # pixels from top
+            color=alt.Color("color", scale=None, legend=None),  # type: ignore
+            x=alt.X("start", axis=alt.Axis(title=text("AXSIS_TIME"))),  # type: ignore
+        )
+    )
+
+    # Einfügen der Phasenbeschriftung
+    text = (
+        alt.Chart(areas_data)  # type: ignore
+        .mark_text(align="left", baseline="middle", dx=7, dy=-135, size=13)  # type: ignore
+        .encode(x="start", x2="stop", text="Phasen+1", color=alt.value("#FAFAFA"))
+    )
+
+    return areas, text
+
+
+def make_simple_plot(chart_data, areas, text):
+    chart = (
+        alt.Chart(chart_data)  # type: ignore
+        .mark_line(color="#FF4B4B")  # type: ignore
+        .encode(x="Zeit", y="Fluoreszenz", tooltip="Fluoreszenz")
+    )
+
+    # coloured areas
+
+    st.altair_chart(areas + chart + text, use_container_width=True)
+
+
+def make_expert_plot(NPQ, tm, PhiPSII, areas, text, left, right):
+    with left:
+        chart_data1 = pd.DataFrame({"NPQ": NPQ, "Zeit": tm / 60})
+
+        chart1 = (
+            alt.Chart(chart_data1)  # type: ignore
+            .mark_line(color="#FF4B4B")  # type: ignore
+            .encode(x=alt.X("Zeit"), tooltip="NPQ", y=alt.Y("NPQ", axis=alt.Axis(title="NPQ [a.u.]")))  # type: ignore
+        )
+
+        points1 = (
+            alt.Chart(chart_data1)  # type: ignore
+            .mark_point(filled=True, size=65, color="#FF4B4B")  # type: ignore
+            .encode(x=alt.X("Zeit"), tooltip="NPQ", y=alt.Y("NPQ", axis=alt.Axis(title="NPQ [a.u.]")))  # type: ignore
+        )
+
+        st.altair_chart(chart1 + text + areas + points1, use_container_width=True)
+
+        # third graph
+    with right:
+        chart_data2 = pd.DataFrame({"Phi": PhiPSII, "Zeit": tm / 60})
+
+        chart2 = (
+            alt.Chart(chart_data2)  # type: ignore
+            .mark_line(color="#FF4B4B")  # type: ignore
+            .encode(x="Zeit", tooltip="Phi", y=alt.Y("Phi", axis=alt.Axis(title="Φ(PSII)")))  # type: ignore
+        )
+
+        points2 = (
+            alt.Chart(chart_data2)  # type: ignore
+            .mark_point(filled=True, size=65, color="#FF4B4B")  # type: ignore
+            .encode(x=alt.X("Zeit"), tooltip="Phi", y=alt.Y("Phi", axis=alt.Axis(title="Φ(PSII)")))  # type: ignore
+        )
+
+        st.altair_chart(chart2 + text + areas + points2, use_container_width=True)
+
+
+# FIXME: version here should probably be replaced by text
+def make_page(text: Callable[[str], str], version: str) -> None:
+    st.markdown(text("HEADLINE_BRAIN"))
+
+    # FIXME: why is col3 unused?
```

Comment:
To make the GIF a little smaller and to make it look the same on all devices, I reduced its size by dividing it into three parts and placed it in the center.